.github/workflows/docker.yml

```diff
@@ -23,7 +26,15 @@ jobs:
             --build-arg BUILDKIT_INLINE_CACHE=1 \
             --cache-from "dependabot/dependabot-core:latest" \
             .
-      - name: Push image to packages
+      - name: Push image to packages (latest)
+        if: '!contains(github.ref, "refs/tags")'
         run: |
           docker login -u ${{ secrets.DOCKER_USERNAME }} -p ${{ secrets.DOCKER_PASSWORD }}
           docker push "dependabot/dependabot-core:latest"
+      - name: Push image to packages (tagged)
+        if: contains(github.ref, "refs/tags")
```

Comment:
Think this is a nice improvement from circle days where we would just keep pushing to the tagged version with any new commits to main

Reply:
👍  It's weird to disconnect the docker image tag from the repo tag, when equivalent values exist:
I expect `dependabot/dependabot-core:0.118.1` to be `refs/tags/v0.118.1`, not `refs/tags/v0.118.2~1`.

I'd dig that for `dependabot/dependabot-core:0.118` and `dependabot/dependabot-core:0` maybe (but YAGNI nobody wants these).